src/frontend/Komponenter/Behandling/Brev/BrevFane.tsx

```diff
@@ -62,6 +67,10 @@ export const BrevFane: React.FC<Props> = ({ behandling }) => {
         settBrevRessurs(respons);
     };
 
+    const åpneUrlINyttVindu = (url: string) => {
+        window.open(url, 'BrevmalerVindu', 'width=1200,height=1000');
```

Comment:
`Using "_blank" as the target attribute value will create several new and unnamed windows on the user's desktop that cannot be recycled or reused. Try to provide a meaningful name to your target attribute and reuse such target attribute on your page so that a click on another link may load the referenced resource in an already created and rendered window` https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Reply:
Usikker på om jeg tenker det er en god idé med "popup-vindu" - da dette ikke er et fullverdig nettleservindu, og skjønner brukerne forskjellen på disse eller vil det skape forvirring?